src/goto-cc/CMakeLists.txt

```diff
@@ -44,32 +44,32 @@ else()
   install(CODE "execute_process( \
     COMMAND ${CMAKE_COMMAND} -E create_symlink \
     goto-cc \
```

Comment:
So aside from the whole `DESTDIR` issue I am learning more about symlinks as well. Didn’t know they worked with relative paths too, though that certainly is very convenient.